randovania/games/am2r/logic_database/Distribution Center.txt

```diff
@@ -1754,6 +1755,10 @@ Extra - minimap_data: [{'x': 67, 'y': 52}, {'x': 67, 'y': 53}, {'x': 68, 'y': 51
 > Horizontal Dock to Spiky Space Jump Trial Entrance; Heals? False
   * Layers: default
   * 3-Tiles High Dock to Spiky Space Jump Trial Entrance/Horizontal Dock to Spiky Space Jump Trial
+  * Extra - facing: left
```

Comment:
Should this not be on the `spiky space jump trial reward` vs the entrance pipe? (assumption based on the below change)

Reply:
good catch